### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by jordandenison**
Refactor codebase for consistency and terseness.

### CODE REVIEW

**Comment by jhabdas** [comment]
Nice brevity, use of reduce over maps and early terminations.

**Comment by jordandenison** [comment]
Thank you!

File: src/lib/config.js [hunk 1]
```diff
@@ -44,13 +44,13 @@ const config = {
    * Checks arguments for specific (immediate action) flags and config
    * @param {Object} args The arguments passed in
    */
-  checkArgs: (args) => {
+  checkArgs: args => {
     // Show help
-    if (args.h) { output.log(config.helpMsg); process.exit(0); }
+    if (args.h) output.log(config.helpMsg); process.exit(0);
```
Comment (TomFrost) [inline src/lib/config.js]: Doesn't this exit the process every time? The conditional only applies to the log; the process.exit will get called every time this runs. Same with the next edit.

Comment (jordandenison) [inline src/lib/config.js]: Ah yes, good catch - they have been restored

**Comment by Fluidbyte** [comment]
nice work!

**Pull request merged**